openmls/src/group/public_group/mod.rs

```diff
@@ -79,7 +79,20 @@ impl PublicGroup {
         group_context: GroupContext,
         initial_confirmation_tag: ConfirmationTag,
     ) -> Self {
-        let interim_transcript_hash = vec![];
+        let backend = OpenMlsRustCrypto::default();
+        let crypto = backend.crypto();
+
+        let interim_transcript_hash = {
+            let input = InterimTranscriptHashInput::from(&initial_confirmation_tag);
+
+            input
+                .calculate_interim_transcript_hash(
+                    crypto,
+                    group_context.ciphersuite(),
+                    group_context.confirmed_transcript_hash(),
+                )
+                .unwrap()
```

Comment:
We will need to handle this properly. As soon as we agree that this is what we want, I can make this method take a `backend` and return a `Result`.